### ISSUE TITLE
RC SE calculation

### ISSUE THREAD

**Issue by andrjohns** [nhs-r-community/FunnelPlotR#30]
With the calculation of the standard errors for ratio of counts data, Spiegelhalter gives the formula:

![s1](https://user-images.githubusercontent.com/27717896/112589118-8295c480-8e3b-11eb-8730-b07e4d37b043.png)

But I see that you're using:

![s2](https://user-images.githubusercontent.com/27717896/112589374-ed470000-8e3b-11eb-822a-fa064ec228ab.png)

I know that the 0.5 is to handle 0 counts, but would you have a brief explainer for the rest? Is this a more robust standard error? Thanks!

**Comment by chrismainey** [comment]
Hi, I've used Spiegelhalter's work as the basis for a lot, but I actually used the implementation of it in guidance used by the UK healthcare regulator, the CQC: https://www.cqc.org.uk/sites/default/files/Intelligent%20Monitoring%20Statistical%20Methodology%20July%20FINAL%20PDF.pdf .
I've not examined the difference particularly, but the rationale is exactly what you've described, it's to handle zeros.  The main point here is converting to z-scores after an approximate normal transformation.  It's to try and allow a fairly standard treatment after that as it's all relative once we are on the z-score scale.
